image_generation/stable_diffusion_1_5/cpp/README.md

```diff
@@ -16,20 +21,22 @@ chmod +x build_dependencies.sh
 "use conda-forge to install OpenVINO Toolkit 2023.1.0 (C++), or download from archives? (yes/no): "
 ```
 
-Notice: Use Intel sample [writeOutputBmp function](https://github.com/openvinotoolkit/openvino/blob/539b5a83ba7fcbbd348e4dc308e4a0f2dee8343c/samples/cpp/common/utils/include/samples/common.hpp#L155) instead of OpenCV for image saving.
+Notice: Use Intel sample 
+[writeOutputBmp function](https://github.com/openvinotoolkit/openvino/blob/539b5a83ba7fcbbd348e4dc308e4a0f2dee8343c/samples/cpp/common/utils/include/samples/common.hpp#L155) instead of OpenCV for image saving.
```

Comment:
It seem you are trying to maintain some line length limit for non-list entries. But this idea is abandoned here

Reply:
Yeah, the line length is not a strict rule :) It makes it easier for some people to read and edit. With this long URL it won't help much, as we don't want to break the url itself. We can line-break the last part.
```suggestion
[writeOutputBmp function](https://github.com/openvinotoolkit/openvino/blob/539b5a83ba7fcbbd348e4dc308e4a0f2dee8343c/samples/cpp/common/utils/include/samples/common.hpp#L155)
instead of OpenCV for image saving.
```